.gitignore

```diff
@@ -22,3 +22,4 @@ stats.json
 
 .env
 .env.*
+/.idea/
```

Comment:
I think `.idea` is a folder from PhpStorm/WebStorm, not VS Code (but I'm not sure which one you use)

If we add it, let's regroup it with similar dev env specificities, like `.vscode` and `.DS_Store` at the top of file and use the same form (simply `.idea`).

Reply:
OK!
I've used both IDEs so I may have mixed up which one produces which folder.